### ISSUE TITLE
Babel works fine for standalone Jest, not within WallabyJS

### ISSUE THREAD

**Issue by kasvtv** [wallabyjs/public#1908]
Within Jest, my babel proposal plugins work fine, but when running through wallabyJS, it seems not to recognize my plugins.

```
​​[Error] Failed to set up project file system: Error: Failed to parse src/entry/entry.test.js, SyntaxError: Unexpected token (5:15)​​
​​[Error]     at _combinedTickCallback (internal/process/next_tick.js:131:7)​​
​​[Error]     at process._tickCallback (internal/process/next_tick.js:180:9)​​

```
Made a simple example repo for it using the optional chaining syntax (containing all my versions as well):
https://github.com/kasvtv/wallaby_issue

Using Ubuntu.

**Comment by smcenlly** [comment]
Thanks for providing the repo! The problem was your compiler file pattern. It was only including `.jsx` files but needs to also include `.js` files. I've created a [pull request](https://github.com/kasvtv/wallaby_issue/pull/1) that fixes this for you.

In your wallaby configuration, I've changed

```diff
...
	compilers: {
-		'**/*.jsx?': wallabyJS.compilers.babel(require('./.babelrc.js'))
+		'**/*.js?(x)': wallabyJS.compilers.babel(require('./.babelrc.js'))
	},
...
```

**Comment by kasvtv** [comment]
Thanks! Thought it was a regex.. I gotta say I feel a little stupid, as the correct syntax appeared elsewhere in the file, hah.

On Nov 8, 2018 23:33, Simon McEnlly <notifications@github.com> wrote:

Thanks for providing the repo! The problem was your compiler file pattern. It was only including .jsx files but needs to also include .js files. I've created a pull request<https://github.com/kasvtv/wallaby_issue/pull/1> that fixes this for you.

In your wallaby configuration, I've changed

...
        compilers: {
-               '**/*.jsx?': wallabyJS.compilers.babel(require('./.babelrc.js'))
+               '**/*.js?(x)': wallabyJS.compilers.babel(require('./.babelrc.js'))
        },
...
